src/lib/components/copy/CopyTemplate.tsx

```diff
@@ -24,15 +24,16 @@ export const CopyTemplate = ({
 
   return (
     <Tooltip isOpen={!isDisabled && hasCopied} label={copyLabel}>
-      <Box
+      <Flex
+        direction="column"
```

Comment:
isn't this the same as `Box`?

Reply:
not exactly, somehow alignment does work only with flex